builder.go

```diff
@@ -94,10 +95,30 @@ func buildAndRunDir(dir string, filters []string, goBuildTags string) error {
 	return nil
 }
 
+func goMod() string {
+	if _, err := os.Stat("go.mod"); os.IsNotExist(err) {
+		return ""
+	}
+	file, err := os.Open("go.mod")
+	defer file.Close()
+	if err != nil {
+		panic(err)
+	}
+	line, err := bufio.NewReader(file).ReadString('\n')
+	if err != nil {
+		panic(err)
+	}
+	return string(line[7 : len(line)-1])
```

Comment:
I think a regex match would be a safer bet here, or, at the very least, a `string.Split()` with a verification of `len(s) > 1 && s[0] == "module"`, otherwise this would panic on an incorrectly formatted mod file in an unfriendly way.

Reply:
went the way of the Split